CHANGELOG.md

```diff
@@ -7,6 +7,13 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Next]
 
+## [2.1.1] - 2022-07-29
+
+### Added
+
+- A warning is emitted if global `log_options` config is set
```

Comment:
I think that this should probably be a "Deprecated" or "Changed" entry. Also it would be nice to have a brief explanation of why it should not be used and what the recommended approach is.